lib/src/components/modal/modal.scss

```diff
@@ -0,0 +1,32 @@
+.ReactModal__Content {
```

Comment:
How comes there are some fixed values in here? (bg colours etc)

Reply:
Great spot, I was meant to convert these back to our hex variables